doc/dev/index.rst

```diff
@@ -744,14 +846,14 @@ reduce the number of tests that are triggered. For instance::
 
 will run as few tests as possible. The tradeoff is that some tests
 will only run on ``ext4`` and not on ``btrfs``, but all files in the
-suite will be in at least one test.
+suite will be in at least one test. FIXME
```

Comment:
s/FIXME/Understanding the actual logic that drives this requires reading the source code./